src/js/constants.js

```diff
@@ -744,7 +744,7 @@ export const EXPLORE_HEADER_ARTICLES_LABELS = {
       <p>Source: OpenAlex</p>"
   },
   "is_oa": {
-    label: "Open Access?",
+    label: "Free-to-read?",
```

Comment:
Wait so we’re switching the labelling of Open Access to Free-to-read for everyone?